html.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = `
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charSet="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>SVG to GeoJSON</title>
+    <link rel="stylesheet" href="https://api.mapbox.com/mapbox-assembly/mbx/v0.20.2/assembly.min.css">
+    <link rel="stylesheet" href="https://api.mapbox.com/mapbox-assembly/mbx/v0.20.2/assembly.min.css">    
```

Comment:
Whoops, looks like line 10 and 11 are duplicates.

Reply:
Ack, good catch!